lib/spotlight/dor/indexer.rb

```diff
@@ -25,7 +25,10 @@ class Indexer < GDor::Indexer
         
       end
     end
-    
+
+    # add a new before_index :method_name and then define the method (example below) to make additions to the spotlight solr doc
+    #before_index :add_something_to_solr_doc
```

Comment:
I guess we should get rubocop + hound hooked up to this project.. I'm pretty sure rubocop would want you to have a space between `#` and `before_index`.